test/unit/specs/FontStyle.spec.js

```diff
@@ -1,38 +1,42 @@
 import Vue from 'vue'
 import FontStyle from '@/components/FontStyle'
 
-// Test number of font colors showing
-describe('FontStyle.vue', () => {
-  it('Font Colors should be 11', () => {
-    const Constructor = Vue.extend(FontStyle)
-    const vm = new Constructor().$mount()
-    expect(vm.$data.fontColors.length).to.equal(11)
+describe('FontStyle.vue', function () {
+  let Constructor, vm
+
+  before(function () {
+    Constructor = Vue.extend(FontStyle)
```

Comment:
The `Constructor` does't need to be hoisted out into the scope of the spec, only the `vm` needs to

Reply:
done